ginisdk/src/main/java/net/gini/android/SdkBuilder.java

```diff
@@ -278,12 +345,57 @@ private synchronized DocumentTaskManager getDocumentTaskManager() {
      * Return the {@link SessionManager} set via #setSessionManager. If no SessionManager has been set, default to
      * {@link AnonymousSessionManager}.
      *
-     * @return                      The SessionManager instance.
+     * @return The SessionManager instance.
      */
     public synchronized SessionManager getSessionManager() {
         if (mSessionManager == null) {
             mSessionManager = new AnonymousSessionManager(mEmailDomain, getUserCenterManager(), getCredentialsStore());
         }
         return mSessionManager;
     }
+
+    /**
+     * Helper method to get local certificates from assets
+     *
+     * @param certFilePaths An array containing all certificates paths relatively to the assets
+     * @throws IOException if the the certificate is not found or it is invalid.
+     * @throws CertificateException if parsing problems are detected when generating Certificate
+     * @return Local certificates
+     */
+
+    private synchronized X509Certificate[] getLocalCertificatesFromAssets(String[] certFilePaths) throws IOException, CertificateException {
+        List<X509Certificate> certificates = new ArrayList<>();
+        AssetManager assetManager = mContext.getAssets();
+        for (String fileName : certFilePaths) {
+            InputStream fis = assetManager.open(fileName);
```

Comment:
as I wrote above, maybe check if it exists and throw something that goes back to the user of the GVL.